src/state/slices/common-selectors.ts

```diff
@@ -229,3 +233,21 @@ export const selectHighestMarketCapFeeAsset = createSelector(
     return sortedFeeAssets[0]
   },
 )
+
+export const selectIsCustomAsset = createSelector(
+  selectAssetById,
+  (_state: ReduxState, assetId: AssetId) => assetId,
+  (asset, _assetId): boolean => {
```

Comment:
suggestion: remove this extra param if possible

```suggestion
  (asset): boolean => {
```

Reply:
Addressed in https://github.com/shapeshift/web/pull/7370/commits/45d970641b800d671d18b0b40d9f616e9afe0f6d